UIDokan/Sources/UIDokan/Extentions/Images+Style.swift

```diff
@@ -0,0 +1,10 @@
+import UIKit.UIImage
```

Comment:
Let's add a comment here to indicate that this is a generated file to avoid any manual update to it.. or we can rename the result file to `UIImage.Generated.swift`

Reply:
Done